.gitignore

```diff
@@ -34,11 +34,11 @@ doc/api/
 node_modules/
 aws-exports.js
 awsconfiguration.json
-amplifyconfiguration.dart
+amplifyconfiguration*.dart
```

Comment:
Q: Why this change?

Reply:
To allow multiple configuration files in your local dev. For example, I've been using a pattern like `amplifyconfiguration.gen1.dart`, `amplifyconfiguration.gen2.dart`, etc.

Reply:
*issue:* There are multiple files committed that match `amplifyconfiguration*.dart` or `amplify_outputs*.dart`. You will either need to update the gitignore (preferably in the relevant packages) to allow those files explicitly or update those file names. Below are the files that I am aware of that match these patterns:
- packages/amplify_core/lib/src/config/amplify_outputs/amplify_outputs.dart
- packages/amplify_core/test/config/amplify_outputs_mapping/data/amplifyconfiguration.g.dart
- packages/amplify_core/test/config/amplify_outputs_mapping/data/amplify_outputs.g.dart

Reply:
I pushed a commit to add gitignore files to allow those files